File: app/src/main/java/com/jeksonshar/jweather/model/WeatherListModel.java
package com.jeksonshar.jweather.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class WeatherListModel {

    @SerializedName("consolidated_weather")
    private List<WeatherModel> consolidatedWeather;

    @SerializedName("title")
    private String title;

    public List<WeatherModel> getConsolidatedWeather() {return consolidatedWeather;}

    public String getTitle() {
        return title;
    }

}

File: app/src/main/java/com/jeksonshar/jweather/model/WeatherListModel.kt
package com.jeksonshar.jweather.model

import com.google.gson.annotations.SerializedName
import com.jeksonshar.jweather.model.WeatherModel
//TODO data class нужно сделать, делать надо val
class WeatherListModel {
    @SerializedName("consolidated_weather")
    val consolidatedWeather: List<WeatherModel>? = null

    @SerializedName("title")
    val title: String? = null
}